### ISSUE TITLE
`(*) *BOLD* text` renders wrong in code-pane

### ISSUE THREAD

**Issue by p-i-** [MacDownApp/macdown#700]
It considers the first two `*`s to be the start and end of bold-text, rather than the second and third.

The preview pane gets it right!

**Comment by franklinyu** [comment]
You mean `` `(*) *BOLD* text` `` or `(*) *BOLD* text`?

**Comment by franklinyu** [comment]
It might be related to [PEG Markdown Highlight](https://github.com/ali-rantakari/peg-markdown-highlight), but I have no idea how to confirm.

**Comment by Jmuccigr** [comment]
Possibly related?: putting a space before the closing `**` does not result in bold text, even though the editor pane highlights it that way.

**Comment by franklinyu** [comment]
@Jmuccigr Yes, and it should not have. Normal user should avoid those corner cases (because they *will* break in some platform, even if not in MacDown), but PEG *is* pretty buggy (or at least non-standard) and we are still looking for alternatives.

**Comment by Jmuccigr** [comment]
@franklinyu, Can I file a bug report somewhere or is it a known one?

And I agree that best practice would be not to do that, but…

**Comment by franklinyu** [comment]
@Jmuccigr I just filed ali-rantakari/peg-markdown-highlight#19. Let's see.

**Comment by Jmuccigr** [comment]
For reference, pandoc will accept `**bold **` as bold unless you specify commonmark as the source.
